src/locales/fr/PAGES/SCOPE.json

```
@@ -0,0 +1,26 @@
+{
+    "TITLE": "Étape 1 : Déterminer le périmètre de l’évaluation",
+    "INTRO": "Définissez les paramètres généraux et le périmètre de l’évaluation. L’idéal est de le faire avec la personne qui a commandé l’évaluation (qui peut ou non être propriétaire du site Web), afin de garantir des attentes communes quant au périmètre de l’évaluation.",
+    "RESOURCE_LINK_NAME": "Étape 1 : Déterminer le périmètre de l’évaluation",
+    "LABEL_SITE_NAME": "Nom du site Web",
+    "INF_SITE_NAME": "Donnez un nom au site Web que vous souhaitez évaluer. Par exemple : <ul><li>« Boutique en ligne de l'entreprise Exemple »</li><li>« Site Web public de l'organisation Exemple »</li><li>« Intranet de l’université Exemple »</li></ul>",
+    "LABEL_SITE_SCOPE": "Périmètre du site Web",
+    "INF_SITE_SCOPE_0": "Définissez le périmètre du site Web de manière à savoir clairement quelles pages Web sont incluses dans l’évaluation. Pour plus d’informations, voir [WCAG-EM Étape 1.a  : Définir le champ d’application du site Web](https://www.w3.org/TR/WCAG-EM/#step1a). Par exemple :",
+    "INF_SITE_SCOPE_LI0": "« Tout le contenu du site Web public de l’organisation Exemple, situé à l’adresse `http://www.example.org` »",
+    "INF_SITE_SCOPE_LI1": "« Tout le contenu Web de la boutique en ligne de l’organisation Exemple, située à l'adresse `http://www.example.org/shop/` »",
+    "INF_SITE_SCOPE_LI2": "« Tout le contenu Web de la version mobile du site Web public de l’organisation Exemple, situé à l'adresse `http://m.example.org` »",
+    "LABEL_WCAG_VERSION": "Version des WCAG",
+    "INFO_WCAG_VERSION": "Sélectionnez la version des WCAG à utiliser. Version 2.1 (par défaut) ou 2.0",
+    "WCAG21": "WCAG 2.1",
+    "WCAG20": "WCAG 2.0",
+    "LABEL_CONFORMANCE_TGT": "Objectif de conformité",
+    "INF_CONF_TGT": "Sélectionnez un niveau de conformité WCAG 2 cible (A, AA ou AAA) pour l’évaluation. Pour plus d’informations, voir [WCAG-EM Étape 1.b : Définir la cible de conformité](https://www.w3.org/TR/WCAG-EM/#step1b). Cette sélection détermine les filtres de niveau de conformité qui sont actifs par défaut à l’« Étape 4  : Auditer l’échantillon ».",
+    "LABEL_SUPPORT_BASE": "Environnement de réference pour évaluer l'accessibilité",
```

```suggestion
    "LABEL_SUPPORT_BASE": "Référence pour la prise en charge de l’accessibilité",
```

@KodGit As discussed together.